nginx_proxy/config.json

```diff
@@ -17,6 +17,10 @@
     "certfile": "fullchain.pem",
     "keyfile": "privkey.pem",
     "hsts": "max-age=31536000; includeSubDomains",
+    "cloudflare": {
+      "enable": false,
+      "real_ip_header": "CF-Connecting-IP"
```

Comment:
Why need that to be a customable value?

Reply:
I was thinking that maybe someone had a need to use x-forwarded-for when multiple connecting proxies was in use, but i also see that its a one-time setting for now, since i dont perform any update on the cloudflare.conf after first time enabled.